src/libstd/f32.rs

```diff
@@ -580,9 +546,9 @@ impl f32 {
     /// # Examples
     ///
     /// ```
-    /// use std::f32;
+    /// use std::f32::consts;
     ///
-    /// let x = f32::consts::FRAC_PI_2;
+    /// let x = consts::FRAC_PI_2;
```

Comment:
I might even personally write `use std::f32::consts as f32` but don't tell anyone that.

Reply:
I completely agree with you. I think the precision should be more obvious. But moving these mathematical consts to associated constants were met with resistance in the RFC (😢).

The reason I went with the way I did was that:
1. `src/libcore/num/{f32,f64}.rs` had examples using this format (`to_degrees` and `to_radians`). So, consistency. Otherwise we should probably change those instances into something else also.
2. Continuing to import the `std::f32` module will shadow the constants on the type, so the constants from the module will still be used. This means it will be even harder to ever argue for a deprecation of the old constants, because it would generate warnings in all such code.

I could do `use std::f32::consts as f32consts;` to avoid all problems. But it's ugly. That's my main problem here. There is no way to do it nicely with the way the constants are currently exposed.

What do you think we should do?

Reply:
Would you be against removing the import and access the constants with fully qualified names?:
```rust
let x = std::f32::consts::FRAC_PI_2;
```

I did that and pushed now. But I have separated these examples using `fxx::consts` to a separate commit. So if you are not convinced I can just remove those commits and we'll leave those examples as they are already, with `use std::f32;`.